manifests/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -0,0 +1,111 @@
+{{if .InstallOVS}}
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovs
+  namespace: openshift-sdn
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the openvswitch daemon.
+spec:
+  selector:
+    matchLabels:
+      app: ovs
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: ovs
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      serviceAccountName: sdn #needed to run privileged pods; not used for api access
+      hostNetwork: true
+      hostPID: true
+      priorityClassName: system-node-critical
+      containers:
+      - name: openvswitch
+        image: {{.Image}}
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
```

Comment:
This all seems kind of terrible... We can't have an actual `.sh` file?

Reply:
copied directly from openshift-ansible!

I think the point is that this can be changed without also taking an openshift PR.